dbms/src/Core/AccurateComparison.h

```diff
@@ -225,6 +225,29 @@ inline bool greaterOp<DB::UInt64, DB::Float32>(DB::UInt64 u, DB::Float32 f)
     return greaterOp(u, static_cast<DB::Float64>(f));
 }
 
+template<>
+inline bool greaterOp<DB::Float64, DB::UInt128>(DB::Float64 f, DB::UInt128 u)
+{
+    return f > u && greaterOp(f, static_cast<UInt64>(u));
```

Comment:
It isn't support because it doesn't have logical sense to compare Float64 (or every numeric type) with an UUID (as same like a Date/DateTime with a numeric type) so we have an `Illegal types of arguments` error if we try to compare an UUID and a numerical type.

But even if the code can't be call using the client, we need to implement it because the compiler require it.